Include/compile.h

```diff
@@ -106,4 +113,7 @@ PyAPI_FUNC(int) _PyAST_Optimize(struct _mod *, PyArena *arena, _PyASTOptimizeSta
 /* This doesn't need to match anything */
 #define Py_fstring_input 800
 
+/* Value of 'optimize' to deactivate all optimizations */
```

Comment:
I would prefer to be consistent: either use "deactive" everywhere, or "disable" everywhere.

```suggestion
/* Value of 'optimize' to disable all optimizations */
```